### PR TITLE
ENH: Use AcquisitionTime for acq_time scans file field

### LINKED ISSUE #450: Use AcquisitionTime for acq_time column of scans tsv files
### Summary

It looks like heudiconv [uses ContentTime](https://github.com/nipy/heudiconv/blob/75f2850f7f07a95ff5875aba1dfd76c09e4be201/heudiconv/bids.py#L408-L411) in the scans tsv files instead of AcquisitionTime, although I think that AcquisitionTime would be more appropriate (i.e., closer to the actual start of the scan).

Quoting @pvelasco in https://github.com/physiopy/phys2bids/pull/219#issuecomment-626899271: 
>>In dicoms, SeriesTime, InstanceCreationTime, and ContentTime are very similar, but AcquisitionTime is generally ~20 seconds earlier than the rest. We need to determine which time is closest to the scanner's trigger pulse.

>From some documentation I found from Siemens on image reconstruction:

>>AcquisitionTime: The Time the acquisition of data started.

>From an answer given by Chris Roden (dcm2niix):

>>Instance Creation Time 0008,0013 initially reflects the time an image leaves a parallel processing reconstructor, so it has a variable delay with regards to the acquisition time. Acquisition Time is more meaningful for data.

>So, if the reconstruction involves something more than a straight FFT (i.e., GRAPPA, SENSE, etc.), that will explain the ~20 sec delay you find between the AcquisitionTime and the other times.
>If I remember correctly, for Siemens scanners, the "scanner trigger" is sent at the time of the excitation pulse of the first slice of each (collected) volume. So the very first trigger will be sent just a few milliseconds before the first chunk of data for the first slice for the first volume is acquired (AcquisitionTime).

This issue is related to #447.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #450. 

Changes proposed:
- Change ContentTime to AcquisitionTime for acq_time field in scans file.
- Change ContentDate to AcquisitionDate for acq_time field in scans file.

### CODE REVIEW

File: heudiconv/bids.py [hunk 1]
```diff
@@ -404,10 +404,10 @@ def get_formatted_scans_key_row(dcm_fn):
     """
     dcm_data = dcm.read_file(dcm_fn, stop_before_pixels=True, force=True)
     # we need to store filenames and acquisition times
-    # parse date and time and get it into isoformat
+    # parse date and time of start of run acquisition and get it into isoformat
     try:
         date = dcm_data.ContentDate
-        time = dcm_data.ContentTime
+        time = dcm_data.AcquisitionTime
```
Comment (yarikoptic) [inline heudiconv/bids.py:410]: is there AcquisitionDate?  very unlikely but still possible that date could be different in some very rare case of scanning around midnight.

Comment (tsalo) [reply heudiconv/bids.py:410]: I _do_ have AcquisitionDate in my dicoms, and the standard has it listed, so I'll switch out ContentDate with AcquisitionDate.

Comment (tsalo) [reply heudiconv/bids.py:410]: Should be good now.

**Comment by yarikoptic** [comment]
Let the heaven not to fall upon our weak heads! ;)

**Comment by yarikoptic** [comment]
Thank you @tsalo !

**Pull request merged**